.github/workflows/pr-welcome-msg.yml

```diff
@@ -21,15 +21,17 @@ jobs:
             ## **Thank you for contributing to the Leapp project!**
             Please note that every PR needs to comply with the [Leapp Guidelines](https://leapp.readthedocs.io/en/latest/contributing.html#) and must pass all tests in order to be mergeable.
             If you want to request a review or rebuild a package in copr, you can use following commands as a comment:
-            - **review please** to notify leapp developers of review request
+            - **review please @oamg/developers** to notify leapp developers of the review request
```

Comment:
@fernflower  I tried to replace the `@` by `<at>` to be sure that people are not pinged with each new welcome msg. Would this lead to such a situation?

Reply:
From what I see in a test run https://github.com/fernflower/leapp-repository/pull/17#issuecomment-1521570808 that's not the case. I'd leave a `@` here and submit a patch later if we are not happy.

Reply:
@fernflower thanks. you are right, it seems ok. let's merge!